harismah/Dwella
# [ERROR DON'T MERGE] Added Unit Tests Configuration and sample test

**Configuration files added**: 
- Added jest.config.js for Jest configuration to customize testing behavior.
- Added babel.config.js to configure Babel for transpiling our JavaScript code.
- Added setupTests.js for setting up the testing environment before running the tests.

**Unit Test:** 
Implemented a sample unit test for Email Verification UI component.

**How to test**:
1. Navigate to the client directory:
   ```bash
   cd client

2. Ensure all dependencies are installed by running:
   ```bash
    npm install
    
3. To run all tests run: 
   ```bash
    npx jest

4. If you want to run specific test file: 
   ```bash
     npx jest src/tests/file.test.jsx
